README.md

```diff
@@ -196,6 +228,8 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Your Answer:**
 
+When I refresh, it sends me back to the the `/login` route. When the page is reloaded I no longer have access to currentUserId.
```

Comment:
This is because the user component is attempting to access the data before it is done loading.